02-tamagotchi-interaction/tests/interaction.rs

```diff
@@ -1,12 +1,58 @@
-use gtest::{Program, System};
-
-// TODO: 0️⃣ Copy tests from the previous lesson and push changes to the master branch
+use gtest::{Program, System, Log};
+use tmg2_io::{Tamagotchi, TmgAction, TmgEvent};
 
+// Yury: this is just a basic test
+// I didn't find if it's possible to manually advance blocks in the mock environment for testing Mood changes
```

Comment:
You may use the [`System::spend_blocks`](https://docs.gear.rs/gtest/struct.System.html#method.spend_blocks) function to skip blocks.